src/array_partition.jl

```diff
@@ -1,3 +1,5 @@
+using SparseArrays
```

Comment:
wrong location for this.

Reply:
I'll fix this and drop 1.9-specific CI once tests pass. It's absurd that they're taking over half an hour

Reply:
Fixed